Client/Frontend/Strings.swift

```diff
@@ -216,12 +198,6 @@ extension String {
                     value: "Set links from websites, emails, and Messages to open automatically in Firefox.",
                     comment: "Description for small home tab banner shown that allows user to switch their default browser to Firefox.",
                     lastUpdated: .unknown)
-                public static let HomeTabBannerButton = MZLocalizedString(
-                    "DefaultBrowserCard.Button.v2",
-                    tableName: "Default Browser",
-                    value: "Learn How",
-                    comment: "Button string to learn how to set your default browser.",
-                    lastUpdated: .unknown)
                 public static let HomeTabBannerCloseAccessibility = MZLocalizedString(
```

Comment:
That one seems to be used in FxNimbus Generated file (only appears once you've built the app)

Reply:
Ah, that explains it (I didn't miss it, and I re-run the script on a machine where I only clone the repo without building it).